src/Microsoft.TemplateEngine.Edge/Template/TemplateCreator.cs

```diff
@@ -56,61 +58,59 @@ public async Task<TemplateCreationResult> InstantiateAsync(ITemplateInfo templat
                 ICreationResult creationResult = null;
                 string targetDir = outputPath ?? _environmentSettings.Host.FileSystem.GetCurrentDirectory();
 
-                try
+                using (Timing.Over(_logger, "Template content generation"))
```

Comment:
nit: To avoid deep nesting and whitespace change to review, 2 things could be done:
1) just do `using var _ = Timing.Over(_logger, "Template content generation")` that would dispose on method exit
2) alternative is to dispose in already existing `finally` block